src/main/java/com/mapper/mapperpractice/service/PostService.java

```diff
@@ -0,0 +1,27 @@
+package com.mapper.mapperpractice.service;
+
+import com.mapper.mapperpractice.entity.Post;
+import com.mapper.mapperpractice.mapper.PostMapper;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service //サービスだよ宣言
+public class PostService {
+    // final　定数
+    private final PostMapper postMapper;
+
+    //コンストラクタインジェクション　呼び出して部品を使えるようにする
+    public PostService(PostMapper postMapper) {
+        this.postMapper = postMapper;
+    }
+
+    public List<Post> getPosts() {
+        List<Post> posts = postMapper.findAll();
+        return posts;
```

Comment:
細かいですが、書き方は揃えておいた方が印象が良くなります！
```suggestion
        return postMapper.findAll();
```

Reply:
こちらもお願いします。
af08b6b915c5f69a12534bb7507d153b95c68f8a